includes/tracking-events.php

```diff
@@ -1,120 +1,87 @@
 <?php
     if(class_exists('WooCommerce')){
-        // /** check certain tracking scenarios **/
         // if visitor is viewing product
-        if(!$this->single_item_tracked && is_product()){
+        if(is_product()){
             $product = $this->resolve_product(get_queried_object_id());
             $product_id = method_exists($product, 'get_id') ? $product->get_id() : $product->id;
             $this->put_event_in_queue(
-                "window.metrilo.viewProduct('"
-                . $product_id
-                . "');"
+                "window.metrilo.viewProduct('$product_id');"
             );
```

Comment:
Can't this be a single line? It goes to the other `put_event_in_queue` invocations as well.